### PR TITLE
Use GitHub actions as CI.

### PR DESCRIPTION
**Pull request by sallner**

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -4,7 +4,6 @@ source = gocept.zestreleaser.customupload
 
 [report]
 precision = 2
-omit = */interfaces.py
```
Comment (icemac) [inline .coveragerc:7]: Having a minimal coverage here, would be nice.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    strategy:
+      matrix:
+        config:
+        # [Python version, tox env]
+        - ["3.8",   "flake8"]
+        - ["2.7",   "py27"]
+        - ["3.6",   "py36"]
+        - ["3.7",   "py37"]
+        - ["3.8",   "py38"]
+        - ["pypy3", "pypy3"]
```
Comment (icemac) [inline .github/workflows/tests.yml:22]: What about PyPy2?

**Review by icemac — APPROVED** [review]
LGTM.

File: .github/workflows/tests.yml (hunk 2 above)
Comment (sallner) [reply .github/workflows/tests.yml:22]: I thought, that was somehow broken on GHA as we did not use it any where.

File: .coveragerc (hunk 1 above)
Comment (sallner) [reply .coveragerc:7]: Done.

File: .github/workflows/tests.yml (hunk 2 above)
Comment (icemac) [reply .github/workflows/tests.yml:22]: I found out recently in another package what PyPy2 is currently not on GHA.

**Pull request merged**